src/Dapr.Client.Grpc/Dapr.Client.Grpc.csproj

```diff
@@ -1,17 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
-
-  <!-- Additional Nuget package properties. -->
   <PropertyGroup>
     <Description>This package contains the reference assemblies for making Dapr client calls using Grpc.</Description>
   </PropertyGroup>
-
+  <ItemGroup>
+    <Protobuf Include="Protos\dapr.proto" GrpcServices="Client" />
```

Comment:
Instead of adding the proto files here, we should sue them the dapr repo. The code from proto files is generated by running tools on the dapr repo and then checked in here.

Reply:
Okey im closing this pr.